### PR TITLE
Support overriding service notifications (#143)

### LINKED ISSUE #143: Option to disable service start/stop notifications
Service start/stop notifications are generating too many emails. I don't want to disable email notifications completely, but only on these actions.

Probably the [best approach](https://serverfault.com/a/658559/65750) seems to be overriding `actionstart` and `actionstop` defined in `/etc/fail2ban/action.d/sendmail-common.conf` with another config `/etc/fail2ban/action.d/sendmail-common.local`:

```
# Override the Fail2Ban defaults in sendmail-common.conf with these entries
[Definition]
# Disable email notifications of jails stopping or starting
actionstart =
actionstop =
```
Or is there some better way supported by this module?

### PR DESCRIPTION
**Pull request by deric**
#### Pull Request (PR) description

Support e-mail notification configuration directly from Puppet/Hiera. 

#### This Pull Request (PR) fixes the following issues

Fixes #143

### CODE REVIEW

File: templates/common/sendmail.conf.epp [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+<% include stdlib %>
+[Definition]
+<% $actions.keys.each |$key| { -%>
+<%= $key %> = <%= $fail2ban::sendmail_actions[$key] %>
+<% } -%>
+
+[Init]
+<% $config.keys.each |$key| { -%>
+<%= $key %> = <%= $fail2ban::sendmail_config[$key] %>
+<% } -%>
```
Comment (bastelfreak) [inline templates/common/sendmail.conf.epp:11]: please fix the missing `\n` at the end of the file

File: templates/common/sendmail.conf.epp [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Override the Fail2Ban defaults in sendmail-common.conf with these entries
+<% include stdlib %>
```
Comment (bastelfreak) [inline templates/common/sendmail.conf.epp:2]: is that include required for anything? I think we can omit it.

**Comment by bastelfreak** [comment]
Hi @deric, thanks for the work! I made a tiny inline comment. Please have a look.

**Comment by deric** [comment]
@bastelfreak Sure thing, all objections should be fixed.

**Comment by deric** [comment]
Unfortunately these changes aren't enough to disable notifications. Notifications are defined in multiple files.
```bash
$ fail2ban-client get sshd action sendmail-buffered actionstart
printf %b "Subject: [Fail2Ban] <name>: started on `uname -n`
From: <sendername> <<sender>>
To: <dest>\n
Hi,\n
The jail <name> has been started successfully.\n
Output will be buffered until <lines> lines are available.\n
Regards,\n
Fail2Ban" | /usr/sbin/sendmail -f <sender> <dest>
```
I'll try to include integration tests in order to ensure that this really works as expected.

**Comment by bastelfreak** [comment]
thanks for the PR!

**Pull request merged**